test/CMakeLists.txt

```diff
@@ -2,38 +2,37 @@
 add_library(catch2 OBJECT catch2/catch2-main.cpp)
 target_include_directories(catch2 PRIVATE ${MPI_CXX_INCLUDE_PATH})
 
-include_directories (${CMAKE_SOURCE_DIR}/include/pomerol)
-
 # Pomerol tests
-set (tests
-HamiltonianTest
-GF1siteTest
-GF2siteTest
-GF4siteTest
-AndersonTest
-AndersonComplexTest
-Anderson2PGFTest
-Vertex4Test
-SusceptibilityTest
+set(tests
+    HamiltonianTest
+    GF1siteTest
+    GF2siteTest
+    GF4siteTest
+    AndersonTest
+    AndersonComplexTest
+    Anderson2PGFTest
+    Vertex4Test
+    SusceptibilityTest
 )
 
-foreach (test ${tests})
+foreach(test ${tests})
     set(test_src ${test}.cpp)
     add_executable(${test} ${test_src})
     set(test_parameters
         ${MPIEXEC_NUMPROC_FLAG} 1
         ${MPIEXEC_PREFLAGS} ${test} ${MPIEXEC_POSTFLAGS})
     add_test(NAME ${test} COMMAND "${MPIEXEC}" ${test_parameters})
-    target_link_libraries(${test} PRIVATE ${MPI_CXX_LIBRARIES} pomerol catch2)
+    target_link_libraries(${test} ${PROJECT_NAME} ${MPI_CXX_LIBRARIES} catch2)
```

Comment:
Out of curiosity, why is PRIVATE gone here?

Reply:
For no reason, I removed it accidentally.

Strictly speaking, the missing keyword is equivalent to `PUBLIC`, which is not what we want here. In practice, however, no target is going to link to the unit tests, so it does not really matter.